src/commit.c

```diff
@@ -283,11 +283,9 @@ int apk_solver_commit_changeset(struct apk_database *db,
 		if (change->old_pkg)
 			size_diff -= change->old_pkg->installed_size / 1024;
 	}
-	size_unit = 'K';
-	if (labs(size_diff) > 10000) {
+	size_unit = apk_get_human_unit(size_diff, NULL);
```

Comment:
This change is untested and I am unsure if it is (a) correct and (b) useful. Which I why I made it in a separate commit.